core/src/main/java/nl/nn/adapterframework/pipes/EtagHandlerPipe.java

```diff
@@ -42,23 +40,23 @@
  *
  */
 public class EtagHandlerPipe extends FixedForwardPipe {
```

Comment:
ik snap eigenlijk niet wat de use case van deze pipe is.

Reply:
Hij was (ooit) bedoeld om de etag uit te kunnen lezen er hem vroegtijdig af te kunnen vangen.
De ApiListenerServlet doet het ook maar dan roept hij wel eerst de adapter aan om te kijken of de etag overeenkomt. (het scheelt dus alleen bandbreedte)

Reply:
Maar hoe dan? en hoe mapt dat op de actions? Wat is het resultaat en wat doe je daar mee?